doc/http.qbk

```diff
@@ -41,76 +41,236 @@ both Boost.Asio and the HTTP protocol specification described in
 
 
 
-[section:example Example]
+[section:usage Usage]
+
+[note
+    Sample code and identifiers mentioned in this section are written
+    as if the following declarations are in effect:
+    ```
+    #include <beast/http.hpp>
+    using namespace beast;
+    ```
+]
+
+A HTTP message (referred to hereafter as "message") contains a request or
```

Comment:
This could go both ways. I've written it out as if the acronym was expanded. e.g.
"A Hypertext Transfer Protocol Message"

Open to change if there is enough demand.